src/SignalRServiceExtension/Config/SignalROptions.cs

```diff
@@ -1,11 +1,14 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using Microsoft.Azure.SignalR.Management;
+
 namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
 {
     public class SignalROptions
     {
         public string ConnectionString { get; set; }
         public string HubName { get; set; }
+        public ServiceTransportType ServiceTransportType { get; set; } = ServiceTransportType.Persistent;
```

Comment:
>Persistent [](start = 94, length = 10)

Shall we use Transient mode as default? Suppose user working on Azure Function would be familar with RestApi and it doesn't have any requirement on function settings, like always on.
Also update docs to explain this configuration.

Reply:
Sounds good. What's more, the deafult mode in management sdk is transient.